### PR TITLE
Sing in up

### PR DESCRIPTION
**Pull request by noklen**

### CODE REVIEW

File: src/app/services/http.service.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ export class HttpService {
 
   constructor (private http: HttpClient) { }
 
-  public get(url: string, api: string = environment.api): Observable<any> {
-    return this.http.get(api + url);
+  public get(url: string, head: object, api: string = environment.api): Observable<any> {
```
Comment (AlexanderBeichuk) [inline src/app/services/http.service.ts:13]: Why we need head?

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <header></header>
-<router-outlet></router-outlet>
+<!--<router-outlet></router-outlet>-->
```
Comment (AlexanderBeichuk) [inline src/app/app.component.html:2]: Why remove router-outlet

File: src/app/services/authentication.service.ts [hunk 3]
```diff
@@ -20,22 +24,49 @@ export class AuthenticationService {
...
+
+  private setCurrentUser(user: User): void {
+    this.currentUser.next(user);
+  }
+
+  private convertTokenToObject(): object {
+    return new HttpParams()
+      .set(BEARER, this.storageService.getStorageBearerToken());
+  }
+
```
Comment (AlexanderBeichuk) [inline src/app/services/authentication.service.ts:69]: We have interceptor for set token to header

Comment (noklen) [reply src/app/services/authentication.service.ts:69]: done

File: src/app/app.component.html (hunk 2 above)
Comment (noklen) [reply src/app/app.component.html:2]: done

File: src/app/services/http.service.ts (hunk 1 above)
Comment (noklen) [reply src/app/services/http.service.ts:13]: done

**Pull request merged**